app/locales/de/messages.po

```diff
@@ -13,1567 +13,1115 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: app/configurator/components/chart-configurator.tsx
 msgid "Add filter"
 msgstr "Filter hinzufügen"
 
```

Comment:
It looks like we lost add string usage location comments? 😱 It's sometimes helpful to see where the string is being used directly in the `po` files, I think you can add them back via `locales:extract` 👍

Reply:
my bad!

Reply:
Thanks!